build.gradle

```diff
@@ -39,12 +39,12 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 10
+    compileSdkVersion 19
```

Comment:
No particular reason, I think at some point while I worked on how I could get the Android emulator testing integration working, it was required by some dependency that is now gone. If the build & tests work fine with version 10 then we can safely revert it back